sql/test_data_second_domain.mysql

```diff
@@ -963,4 +963,4 @@ INSERT INTO civicrm_navigation
 VALUES
     ( @domainID, CONCAT('civicrm/report/instance/', @instanceID,'&reset=1'), 'Mailing Detail Report', 'Mailing Detail Report', 'administer CiviMail', 'OR', @reportlastID, '1', NULL, @instanceID+2 );
 UPDATE civicrm_report_instance SET navigation_id = LAST_INSERT_ID() WHERE id = @instanceID;
-UPDATE civicrm_domain SET version = '4.6.alpha1';
+UPDATE civicrm_domain SET version = '5.27.alpha1';
```

Comment:
@seamuslee001 I tackled this in https://github.com/civicrm/civicrm-core/pull/17729/commits/e37554a6366a6470ba27fe1eb13fa055a5af57e0

Reply:
@seamuslee001 and @totten tackled it in #17734